src/Service/StorageApiService.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Keboola\DockerBundle\Service;
+
+use Keboola\DockerBundle\Exception\ApplicationException;
+use Keboola\DockerBundle\Exception\NoRequestException;
+use Keboola\DockerBundle\Exception\UserException;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Exception;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class StorageApiService extends \Keboola\Syrup\Service\StorageApi\StorageApiService
```

Comment:
proč je tahle servisa "tlustší" než byla před tím?

Reply:
good point! puvodne jsem ji chtel udelat nezavislou na syrupu, jenze to nejde protoze musi byt odvozena od syrupove kvuli tomuhle https://github.com/keboola/syrup/blob/master/src/Keboola/Syrup/Monolog/Handler/StorageApiHandler.php#L28 - tohle teda jeste predelam!